packages/quantic/docs/adding-tests.md

```diff
@@ -0,0 +1,148 @@
+# Add Tests
+
+This article uses an example to guide you through the steps required to create tests for a component.
+
+Let's say you just finished coding a `quantic-greeting` component, which displays a greeting message and accepts a name as an option. For example:
+
+```html
+<c-quantic-greeting name="Alex"></c-quantic-greeting>
+
+<!-- Which outputs: Hello Alex! -->
+```
+
+## Update the Quantic Examples Community
+
+Now, say that you wish to make this component available in the `Quantic Examples` community. Further, say you want to add end to end tests to prevent any regression. To achieve that, you want a community page containing your example component, and a configuration panel allowing users to modify the `name` option.
```

Comment:
```suggestion
Now, say that you wish to make this component available in the `Quantic Examples` community. Further, say you want to add end to end tests to prevent any regression. To achieve that, you want a community page containing your example component, and a configuration panel allowing users to modify the component's options. In this case, you want to test the `name` option.
```

Reply:
I think it just adds precision so we don't start looking for the `name` options on every component 😅